base/partr.jl

```diff
@@ -20,7 +20,56 @@ const heaps = [Vector{taskheap}(undef, 0), Vector{taskheap}(undef, 0)]
 const heaps_lock = [SpinLock(), SpinLock()]
 
 
-cong(max::UInt32) = iszero(max) ? UInt32(0) : ccall(:jl_rand_ptls, UInt32, (UInt32,), max) + UInt32(1)
```

Comment:
Should https://github.com/JuliaLang/julia/blob/0a26e908e77f147a5a964735ad02f74d047fe47b/src/scheduler.c#L90-L94 be deleted, as suggested by https://github.com/JuliaLang/julia/pull/50203/files#diff-7ba7e0372b2249c6b36e693fccd2daca24190ef3001e6e61ac625c02b4bae4d2R91?

Reply:
Bump